contents/blog/introducing-housewatch.md

```diff
@@ -0,0 +1,114 @@
+---
+date: 2023-06-12
+title: "Introducing HouseWatch: An open source toolkit for ClickHouse"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+author:
+  - ian-vanagas
+  - yakko-majuri
+tags:
+  - Product updates
+  - Release notes
+  - ClickHouse
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+---
+
+We are big fans of ClickHouse, we rely on it heavily to store and retrieve the massive amount of data we process every day. In doing this at scale for multiple years now, we’ve built a lot of expertise and systems related to ClickHouse. 
+
+To formalize and share these, we’ve recently built and launched [HouseWatch](https://github.com/PostHog/HouseWatch), an open-source suite of tools for monitoring and managing ClickHouse. HouseWatch is free and works with your existing ClickHouse instance. You can [clone it from GitHub](https://github.com/PostHog/HouseWatch) and deploy it via Docker Compose.
+
+## Why we built HouseWatch
+
+We have used ClickHouse at PostHog since August 2021, when we [moved from Postgres](/blog/how-we-turned-clickhouse-into-our-eventmansion). During this time, we’ve gained a lot of expertise using and debugging ClickHouse. 
+
+ClickHouse provides tons of easily queryable metadata about the system, but knowing how and what to query is difficult. From our usage of ClickHouse, we’ve built an intuition for this, some of which we’ve documented in our [ClickHouse manual](/handbook/engineering/clickhouse). 
+
+We’ve also built a lot of systems and processes for managing clusters. These include:
+
+- Tracking metrics via Grafana
+- Querying via Metabase
+- Running operations on nodes through `ssh`
+- Managing async migrations with [custom-built tools](/blog/async-migrations)
+
+To share our expertise, formalize these tools, and centralize them in one place, Li, Yakko, and Tim built the first version of HouseWatch at our [Aruba offsite](/blog/aruba-hackathon). We felt it would be useful to us, and support other ClickHouse users as well.
+
+## Features
+
+HouseWatch provides multiple battle-tested tools for monitoring and managing ClickHouse.
+
+### Query performance and analysis
+
+To help understand the performance of all the queries on your ClickHouse clusters, we list queries, with emphasis placed on active and slow queries. 
```

Comment:
something more like this might be better?

```suggestion
To help understand the performance of all the queries on your ClickHouse clusters, we provide a list of normalized queries and their performance metrics. You can sort queries in your cluster by average duration, memory usage, and % of IOPS, for example.
```

Reply:
Oh lol I see it below now - but "we list queries" just sounded a bit off